install_config/router/default_haproxy_router.adoc

```diff
@@ -166,6 +169,66 @@ $ oc adm router --dry-run -o yaml --service-account=router
 ----
 endif::[]
 
+[[deploy-router-access-logs]]
+Configuring the Router to Forward HAProxy Logs::
+
+You can configure the router to forward logs generated by HAProxy 
+to an rsyslog sidecar container. The `--extended-logging=true` parameter 
+appends the syslog container to forward HAProxy logs to standard output. 
+
+----
+$ oc adm router --extended-logging=true
+----
+
+For example, the following is the configuration for a router 
```

Comment:
I'd s/For example, the following/The following example

> :no_entry_sign: following n
Do not use as a noun except to refer to a group of followers, adherents, or partisans. For example, do not
write “Do the following” or “The functions include the following” as lead-in wording to a list. Use
“following” only as an adjective, as in “Complete the following steps” or “The adapter has the following
functions.”